salt/environment_settings.yml

```diff
@@ -197,6 +205,19 @@ environments:
           edx-worker:
             number: 1
             type: t2.large
+      sandbox:
```

Comment:
Should this be designated as "sandbox" or as "mitxpro-ci"? Not sure if that impacts how it will function as far as deployment, but semantically this is technically for the CI app so it might be useful and less confusing in the future to make sure the name reflects that.

Reply:
I kinda like this mapping:
ci - sandbox
rc - qa
production (or no env) - production (or no env)

Besides we already have the discrepancy between rc/qa.

Reply:
Fair enough